### PR TITLE
docs(support): jira changes

### PR DESCRIPTION
**Pull request by garima-camunda**
Done changes in https://docs.camunda.org/enterprise/support/#how-to-create-a-support-issue as after 11th Feb JIRA update, UI and some of the features will change for customers.

I will merge the changes and push the build on Monday, 12th Feb after JIRA update is successful. 

@tasso94 - Please feel free to review it.

### CODE REVIEW

File: enterprise/content/support.md [hunk 1]
```diff
@@ -18,17 +18,29 @@ This guide is intended to help you get the support you need quickly and convenie
...
 * Go to our [JIRA support system](https://jira.camunda.com/projects/SUPPORT/queues)
-* Select `Create Issue`
-* Select `Support` as **project** and choose the desired [issue type]({{< relref "#issue-types" >}}).
-* Give the ticket a meaningful **summary**
-* You can give your ticket a [**priority**]({{< relref "#priority-level" >}}) depending on how urgent the issue is for you.
+* Select `Camunda Enterprise Support`
+{{< img src="../img/jira-support-issue.png" title="Create Support Issue" >}}
+* Choose the desired [issue type]({{< relref "#issue-types" >}}).
+{{< img src="../img/jira-select-issue-type.png" title="Select Issue Type" >}}
+* Help Request page looks like as follows:
```
Comment (tasso94) [inline enterprise/content/support.md:26]: 🔧 I would either say "looks like this" or "looks as follows" but not "looks like as follows".

```suggestion
* Help Request page looks as follows:
```

File: enterprise/content/support.md [hunk 2]
```diff
@@ -18,17 +18,29 @@ This guide is intended to help you get the support you need quickly and convenie
...
+{{< img src="../img/jira-help-request-create-issue.png" title="Create Bug Report" >}}
+* Bug Report page looks like as follows:
```
Comment (tasso94) [inline enterprise/content/support.md:28]: 🔧 I would either say "looks like this" or "looks as follows" but not "looks like as follows".

```suggestion
* Bug Report page looks like as follows:
```

File: enterprise/content/support.md [hunk 3]
```diff
@@ -18,17 +18,29 @@ This guide is intended to help you get the support you need quickly and convenie
...
+{{< img src="../img/jira-bug-report-create-issue.png" title="Create Bug Report" >}}
+* Give the ticket a meaningful **summary**.
```
Comment (tasso94) [inline enterprise/content/support.md:30]:
```suggestion
* Provide a meaningful **summary** to the ticket.
```

File: enterprise/content/support.md [hunk 4]
```diff
@@ -18,17 +18,29 @@ This guide is intended to help you get the support you need quickly and convenie
...
+* You can give your ticket a [**priority**]({{< relref "#priority-level" >}}) depending on the impact of an issue on your system.
```
Comment (tasso94) [inline enterprise/content/support.md:31]:
```suggestion
* You can assign a [**priority**]({{< relref "#priority-level" >}}) to your ticket, depending on the impact an issue causes on your system.
```

File: enterprise/content/support.md [hunk 5]
```diff
@@ -18,17 +18,29 @@ This guide is intended to help you get the support you need quickly and convenie
...
 * Provide us with a date in case of important **deadlines** that need to be reached.
-* Enter a detailed **description** of your issue. Make use of the provided template in the description filed (Reproduce steps, Problem, Expected behavior, etc.). In order to represent code snippets in a more readable way use the [JIRA text formatting]({{< relref "#text-formatting" >}}).
+* Enter a detailed **description** of your issue. Make use of the provided template in the description filed (Reproduce steps, Problem, Expected behavior, etc.).
```
Comment (tasso94) [inline enterprise/content/support.md:33]:
```suggestion
* Enter a detailed **description** of your issue. Use the provided template in the description field (Steps to reproduce, Problem, Expected behavior, etc.). 
```

File: enterprise/content/support.md [hunk 6]
```diff
@@ -18,17 +18,29 @@ This guide is intended to help you get the support you need quickly and convenie
...
-* Enter a detailed **description** of your issue. Make use of the provided template in the description filed (Reproduce steps, Problem, Expected behavior, etc.). In order to represent code snippets in a more readable way use the [JIRA text formatting]({{< relref "#text-formatting" >}}).
+* Enter a detailed **description** of your issue. Make use of the provided template in the description filed (Reproduce steps, Problem, Expected behavior, etc.). 
 * **Attachments** like screenshots, log files, stack traces or BPMN files help us to find and reproduce your bug faster. If possible provide log files and stack traces as plain text file.
-* For bug reports and help requests, it is important to know which **environment** you use. Inform us about the used Camunda Platform version, application server, database and everything else that seems to be important for your issue.
+* For bug reports and help requests, it is important to know which **environment** you use. Inform us about the used Camunda version, application server, database and everything else that seems to be important for your issue.
+
+
+# Changes in  JIRA
```
Comment (tasso94) [inline enterprise/content/support.md:38]:
```suggestion
## Changes in  JIRA 
```

File: enterprise/content/support.md [hunk 7]
```diff
@@ -18,17 +18,29 @@ This guide is intended to help you get the support you need quickly and convenie
...
 * **Attachments** like screenshots, log files, stack traces or BPMN files help us to find and reproduce your bug faster. If possible provide log files and stack traces as plain text file.
-* For bug reports and help requests, it is important to know which **environment** you use. Inform us about the used Camunda Platform version, application server, database and everything else that seems to be important for your issue.
+* For bug reports and help requests, it is important to know which **environment** you use. Inform us about the used Camunda version, application server, database and everything else that seems to be important for your issue.
+
+
+# Changes in  JIRA 
+* UI(User Interface) looks different
+* For traceability reasons, comments will not be editable and uploaded attachments can not be deleted. You can request an assigned support agent to update/delete if needed.
+* Comments will appear in reverse chronological order (newest on top)
+* Priority, Description and Environment will be editable via “Edit” link on the right side of the page
```
Comment (tasso94) [inline enterprise/content/support.md:42]: 🔧 According to our writing best practices, we highlight UI elements with bold formatting.
```suggestion
* Priority, Description and Environment will be editable via the **Edit** link on the right side of the page
```

File: enterprise/content/support.md [hunk 8]
```diff
@@ -18,17 +18,29 @@ This guide is intended to help you get the support you need quickly and convenie
...
+* You might need to refresh the page in case any of the standard fields (Assignee, Priority etc.) are not displayed immediately after creating the ticket.
```
Comment (tasso94) [inline enterprise/content/support.md:43]:
```suggestion
* You might need to refresh the page if any standard fields (Assignee, Priority, etc.) are not displayed immediately after creating the ticket.
```

**Review by tasso94 — APPROVED** [review]
Looks almost good to me. 🙂 👍  
Just some optional hints on the wording. Feel free to ignore them if you disagree.

**Comment by garima-camunda** [comment]
Thanks @tasso94 for the quick review. All suggestions look good to me. I just committed the suggestions.

**Pull request merged**